### PR TITLE
LIVE-2848: update letters styling

### PR DESCRIPTION
**Pull request by joecowton1**
## Why are you doing this?

Headline styling was broken on the Letters template, standfirst styling was broken for both letters and analysis.

Notes: 
- there is no 400 weight headline style in Source for the Letters headline, so have gone with 300.
- think the CI errors will probably be fixed by: https://github.com/guardian/apps-rendering/pull/1408#pullrequestreview-715711646 - does this sound correct @faresite ?

## Screenshots

| Before | After |
| --- | --- |
|  |  |
|  |  |

### CODE REVIEW

**Review by abeddow91 — COMMENTED** [review]
It looks like all of the snapshots have been deleted (or a good amount at least). Any reason for this? PR doesn't look like it should be doing that..

File: src/components/editions/headline/index.tsx [hunk 1]
```diff
@@ -42,6 +42,11 @@ const commentStyles = css`
 	padding-right: 6rem;
 `;
 
+const lStyles = css`
+	padding-bottom: 0;
+	padding-right: 6rem;
+`;
+
```
Comment (abeddow91) [inline src/components/editions/headline/index.tsx:49]: Doesn't look like this is being used, can we remove it?

Comment (joecowton1) [reply src/components/editions/headline/index.tsx]: Good spot 👍

**Review by abeddow91 — APPROVED** [review]
Looks good to go!

**Pull request merged**